Util/NHibernate.Caches.Util.JsonSerializer/NHibernate.Caches.Util.JsonSerializer.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../NHibernate.Caches.props" />
+  <PropertyGroup>
+    <Product>NHibernate.Caches.Util.JsonSerializer</Product>
+    <Title>NHibernate.Caches.Util.JsonSerializer</Title>
+    <Description>Json.NET serializer for distributed NHibernate.Caches providers.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
This library may be referenced by .Net Framework applications using old tooling. For best compatibility, better add multi-targeting here, including `net461`.

Reply:
Added.